caas/kubernetes/provider/k8s.go

```diff
@@ -1920,7 +1920,7 @@ func (k *kubernetesClient) ExposeService(appName string, resourceTags map[string
 
 	// TODO(caas): refactor juju expose to solve potential conflict with ingress definition in podspec.
 	// https://bugs.launchpad.net/juju/+bug/1854123
-	_, err = k.ensureIngress(appName, spec, true)
+	_, err = k.ensureIngressV1beta1(appName, spec, true)
```

Comment:
This is internal to ExposeService() so wondering why we use the beta1 api and not v1